### PR TITLE
~Apr 13: Java agent: user tracking [DO NOT MERGE UNTIL 8.1 AGENT RELEASE]

### PR DESCRIPTION
**Pull request by obenkenobi**
To go out with https://github.com/newrelic/docs-website/pull/12265. Estimated release date April 13. Confirm with agent team to publish. 

Adds documentation for user tracking for the Java Agent.

### CODE REVIEW

File: src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx [hunk 1]
```diff
@@ -126,6 +126,15 @@ To instrument [`Transactions`](/docs/apm/applications-menu/monitoring/transactio
         [`NewRelic.ignoreApdex()`](http://newrelic.github.io/java-agent-api/javadoc/index.html?com/newrelic/api/agent/NewRelic.html)
       </td>
     </tr>
+    <tr>
+      <td>
+        To associate a user ID with the current transaction and any errors noticed during the transaction's lifespan by setting an `enduser.id` agent attribute.
```
Comment (zuluecho9) [inline src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx:131]: I find this a little hard to parse; hard to tell what clause applied to what clause; could you rework it a little bit?

**Review by zuluecho9 — DISMISSED** [review]

File: src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx (hunk 1 above)
Comment (obenkenobi) [reply src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx:131]: How does this sound:

Comment (obenkenobi) [reply src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx:131]: How does this sound?

Adds user tracking to the current transaction by setting the `enduser.id` attribute.

**Comment by obenkenobi** [comment]
Closing pull request as it will be replaced with an errors inbox PR
